library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx942/GridBased/aquavanjaram_Cijk_Ailk_Bljk_BBS_BH_Bias_AS_SAV_UserArgs.yaml

```diff
@@ -526692,6 +527210,10 @@
     - [1780, 0.0]
   - - [131072, 2816, 1, 4096]
     - [1780, 0.0]
+  - - [67108864, 32, 1, 32]
+    - [1782, 0.0]
+  - - [67108864, 32, 1, 32]
```

Comment:
why same sizes for 2 solutions?

Reply:
After offline sync, GridBased yaml should be one size with one solution, I will remove another one.

Reply:
Done.